packages/chain-adapters/src/cosmossdk/CosmosSdkBaseAdapter.ts

```diff
@@ -281,4 +297,24 @@ export abstract class CosmosSdkBaseAdapter<T extends CosmosChainTypes> implement
   closeTxs(): void {
     this.providers.ws.close('txs')
   }
+
+  async getValidators(): Promise<Array<chainAdapters.cosmos.Validator>> {
+    try {
+      const { data } = await this.providers.http.getValidators()
+      return data.validators.map<chainAdapters.cosmos.Validator>((validator) =>
+        transformValidator(validator)
```

Comment:
Nitpick: You can call this with `map(transformValidator)` and it will implicitly pass `validator` as an argument so there's no additional arrow function in the stack trace